### ISSUE TITLE
Minor issues

### ISSUE THREAD

**Issue by next2non** [dperson/emby#1]
Hi

Please check these issues I have encountered from my side
1. No restart button next to the shutdown button
2. Cannot enter into edit image of the folders
3. After installing plugin "iptv" there is not option to add links

Regards
Tariq

**Comment by dperson** [comment]
Hi @next2non,
1. On the [forums](http://emby.media/community/index.php?/topic/18819-server-restart-button-in-dashboard/), they seem to say that the restart button might be available on the windows version of the emby server. This is a Linux container, so no dice.
2. I'm not sure that I follow the problem you're having on this one...
3. I haven't used the IPTV plugin yet (in looking at this issue I'm interested in doing so), but it sounds like a restart is needed after installation of the plugin. Have you done a `sudo docker restart <name_of_emby_containter>`?

**Comment by next2non** [comment]
Hi @deperson

I downloaded another container (emby official) and after running it I do see the restart button next to shutdown.

As for image, what I meant that u could assign different images to different libraries. But managed to make it work by exiting the "Library" tab and coming back to it and I was able to edit the Library image for each library

The IPTV plugin after playing with it managed to get it working. The only feature would be to add bulk links rather than one at a time 

Thanks for the reply
